src/shared/persistence/scope-base.ts

```diff
@@ -56,4 +66,19 @@ export abstract class ScopeBase<T extends AnyEntity> {
 
         return this;
     }
+
+    public findBySubstring(
+        fields: Array<keyof T>,
+        substring: string,
+        operator: ScopeOperator = ScopeOperator.OR,
+    ): this {
+        const likeConditions: QBFilterQuery<T>[] = fields.map((field: keyof T): QBFilterQuery<T> => {
+            return { [field]: { $like: `%${substring}%` } };
```

Comment:
do we may search with $ilike instead of $like?

Reply:
Yes makes sense I will adapt it